examples/crm/src/misc/getContactAvatar.ts

```diff
@@ -0,0 +1,75 @@
+import { Contact } from '../types';
+
+export async function hash(string: string) {
+    const utf8 = new TextEncoder().encode(string);
+    const hashBuffer = await crypto.subtle.digest('SHA-256', utf8);
+    const hashArray = Array.from(new Uint8Array(hashBuffer));
+    const hashHex = hashArray
+        .map(bytes => bytes.toString(16).padStart(2, '0'))
+        .join('');
+    return hashHex;
+}
+
+const DOMAINS_NOT_SUPPORTING_FAVICON = [
+    'google.com',
```

Comment:
I'm not sure that the `google.com` is public mail, IMHO it is only used by googlers.